pkg/smart/scsiinquirypage.go

```diff
@@ -57,7 +57,10 @@ func (d *SCSIDev) scsiInquiry() (InquiryResponse, error) {
 // command using Inquiry response struct
 func (inquiry InquiryResponse) getValue() map[string]string {
 	InqRespMap := make(map[string]string)
-	InqRespMap[SPCVersion] = fmt.Sprintf("%.d", inquiry.Version)
+
+	SPCVersionValue := fmt.Sprintf("%.d", (inquiry.Version - 0x02))
```

Comment:
we can avoid this extra variable

Reply:
I thought getting the value and concatenating it should be done separately !

Reply:
if `SPC-` is constant for all the disks then we can remove extra variable else it is good.